konbata.example.yml

```diff
@@ -1,2 +1,7 @@
 # Generally looks like /Applications/LibreOffice.app/Contents/MacOS/soffice
 :libre_office_path: <LibreOffice Path>
+
+:canvas_url: <canvas instance api url>
+:canvas_token: <canvas token>
+:account_id: <account id>
+:request_timeout: <request timeout seconds>
```

Comment:
I think it would be helpful to add some comments to each of these lines giving an example of what the user should be adding. I, for example, wasn't sure what to add for my account ID. After looking at the Senkyoshi.yml file, I saw that I can just use `:default`. I'm also not sure what a sensible request timeout would be.